docs/01-overview/main-areas/telemetry/telemetry-03-traces.md

```diff
@@ -170,10 +188,74 @@ stringData:
   password: XXX
 ```
 
-### Step 5: Rotate the Secret
+### Step 6: Rotate the Secret
 
 As used in the previous step, a Secret referenced with the **secretKeyRef** construct can be rotated manually or automatically. For automatic rotation, update the Secret's actual values and keep the Secret's keys stable. TracePipeline watches the referenced Secrets and detects changes, so the Secret rotation takes immediate effect. When using a Secret owned by the [SAP BTP Operator](https://github.com/SAP/sap-btp-service-operator), you can configure `credentialsRotationPolicy` with a specific `rotationFrequency` to achieve an automated rotation.
 
+## Kyma Components with tracing capabilities
+
+Kyma bundles several modules which are potentially involved in user flows and with must propagate the trace context properly and optional provide custom spans.
+
+### Istio
+
+The Istio module plays a crucial aspect in a distributed trace as the module provides the ingress gateway where usually requests from external are entering the cluster scope and will be enriched with trace context if that did not happen yet. Furthermore, every components being part of the Istio Service Mesh is running an Istio Proxies which propagates the context properly but also creates span data. Having Istio tracing activated and doing trace propagation in your application will already assure that you will get a complete picture of a trace as every component will contribute span data automatically.
+
+The Istio module is configured with an [ExtensionProvider](https://istio.io/latest/docs/tasks/observability/telemetry/) called `kyma-traces`. The provider can be activated on global mesh label using the Istio [Telemetry API](https://istio.io/latest/docs/reference/config/telemetry/#Tracing) by placing a resource to the istio-system namespace like that:
```

Comment:
ExtensionProvider is this how it's written? I don't see any example in the linked doc. Maybe an `extension provider` would be sufficient?

Reply:
```suggestion
The Istio module is configured with an [extension provider](https://istio.io/latest/docs/tasks/observability/telemetry/) called `kyma-traces`. The provider can be activated on global mesh label using the Istio [Telemetry API](https://istio.io/latest/docs/reference/config/telemetry/#Tracing) by placing a resource to the istio-system namespace like that:
```